README.md

```diff
@@ -27,3 +27,15 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+Answer: 
+Invariant Definition: Looking back at the slides. The invariant or proving an invariant is we have to prove the invariant exists before the loop begins. Then at the end of the iteration prove that it holds the value from the previous iteration. 
+
+Within my code, the invariant would be fib(a-1). The reason is that the recursive sequence will go up to a - 1. Let's say we do fib(5) which will return us [0,1,1,2,3,5] as the final result even though you see 6 values within the actual array. This went through the sequence 4 times. Before each recursive call that is being done, it will return the value up to fib(a-1) within the sequence. Then after the recursive call, it will add the new Fibonacci number into the array which is the sum of the last two. With this, the array will hold up to the value of an in the Fibonacci sequence. This also enables it to grow with a. 
```

Comment:
Still not there. The invariant states a property that is true at the beginning of an iteration. `fib(a-1` just is, it can't be true or false.